frontend/src/components/EventsList.tsx

```diff
@@ -0,0 +1,23 @@
+export interface EventItem {
+	title: string;
+	ticketsURL: string;
```

Comment:
Probably worth making this optional, and default to "#" or none if not present.....

Reply:
it should be mandatory, at least for charity events, and be explicitly set to hash or blank rather than inferred if not present

Reply:
Ok, in which case (i'll do this possibly):
add a enum value called type, values `house`, `charity` or something more descriptive
set where it is there or not based on `type`